### DISCUSSION TITLE
START GCODE AND END GCODE

### DISCUSSION THREAD

**Discussion by mustytechy** [mriscoc/Ender3V2S1#287] [Q&A]
hello guys, please guy newbie is here im using cura with my ender 3 s1  with professional firmware, and when i add my printer to the cure as ender 3  as S1 is not on the list, so i was wondering if anything have start and end gcode that work well with the printer please and thanks in advance.

**Comment by carnaue** [comment]
Go to printables.com

Search for:
Adding the Ender 3 S1 printer profile in Cura 

Follow those instructions. 

Gwynn

**Comment by mriscoc** [comment]
Please, search in the Wiki
https://github.com/mriscoc/Ender3V2S1/wiki/Slicer-G-code-Scripts
